dashboards/victoriametrics.json

```diff
@@ -1264,63 +1264,53 @@
             "h": 8,
             "w": 12,
             "x": 12,
-            "y": 36
+            "y": 3
           },
           "hiddenSeries": false,
-          "id": 34,
+          "id": 73,
           "legend": {
-            "avg": false,
-            "current": false,
+            "alignAsTable": true,
+            "avg": true,
```

Comment:
Avg of display range probably isn't useful, given that the query itself is over 1d range.

I'd suggest disabling the legend altogether.

Reply:
Good point!